Cargo.toml

```diff
@@ -158,6 +158,7 @@ atty = "0.2.11"
 backoff = "0.4.0"
 base64 = "0.22.0"
 bincode = "1.3.3"
+bitcoin = "0.31.1"
```

Comment:
can you use the existing bip32 and base58 dependencies instead of introducing a new one?

Reply:
Yes. I did not see the already existing `ed25519-dalek-bip32` dependency, thanks for pointing that out.